tests/dragonfly/tls_replication_test.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import asyncio
+import redis
+from redis import asyncio as aioredis
+from . import DflyInstanceFactory, dfly_args
+import logging
+import time
+from dragonfly.replication_test import check_all_replicas_finished
+
+BASE_PORT = 1111
+ADMIN_PORT = 1211
+
+"""
+Test replication with tls and non-tls options
```

Comment:
Actually this is just another replication test.. you can just move it to replicaiton_test.py